packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -545,9 +552,7 @@ Possible alternatives for such technical user authentication are BasicAuthentica
       case 'on-premise':
         return addProxyConfigurationOnPrem(
           destination,
-          this.subscriberToken?.type === 'iss'
-            ? this.subscriberToken.serviceJwt
-            : this.subscriberToken?.userJwt
+          this.subscriberToken?.userJwt || this.subscriberToken?.serviceJwt
```

Comment:
@FrankEssenberger I lack a bit of knowledge here - what is this used for and is the new implementation correct?

Reply:
Also: Why would we ever pass a service token here?